runtime/php74/config/bref-opcache.ini

```diff
@@ -1,12 +1,4 @@
-; On the CLI we want errors to be sent to stdout -> those will end up in CloudWatch
-display_errors=1
-
-; Since PHP 7.4 the default value is E_ALL
-; We override it to set the recommended configuration value for production.
-; See https://github.com/php/php-src/blob/d91abf76e01a3c39424e8192ad049f473f900936/php.ini-production#L463
-error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT
-
-memory_limit=3008M
+zend_extension=/opt/php-modules/opcache.so
```

Comment:
PHP has an order of folder scan to load extensions. I believe we can still make it work with just the extension name here, but  during iteractions of it I found it much easier to debug if I know exactly which folder was expected to have the extension and why it crashed / was not loaded.
Let me know if I should remove the full path and leave just the relative path

Reply:
Sorry, I just read your comment again and I noticed I didn't fully address it.

So, when we were compiling PHP, we could choose some paths that would be available for scanning PHP extensions. This is no longer true for installing from a distribution. Remi Collet and Amazon Linux have different configuration betweem themselves (by default), so that would mean we would need different config files for ARM and x86. Furthermore, Amazon Linux places extension files under `/etc/...`  and we can't overwrite that from AWS Lambda.

I did replace this with ` /opt/php-extensions/`.